plugins/client/graphql-kotlin-client-generator/src/main/kotlin/com/expediagroup/graphql/plugin/client/generator/types/generateGraphQLEnumTypeSpec.kt

```diff
@@ -51,7 +51,7 @@ internal fun generateGraphQLEnumTypeSpec(context: GraphQLClientGeneratorContext,
     }
 
     val unkownTypeSpec = TypeSpec.anonymousClassBuilder()
-        .addKdoc("This is a default enum value that will be used when attempting to deserialize unknown value.")
+        .addKdoc("%L", "This is a default enum value that will be used when attempting to deserialize unknown value.")
```

Comment:
since the passed in text is a constant without any special chars we technically don't need the extra formatting here

Reply:
Fair enough. I figured the consistency was more important.